pages/home.ts

```diff
@@ -23,8 +23,21 @@ const clickloginButton = async (page: Page, isMobile) => {
     await waitForVisible(page, 'authentication-modal-header');
 }
 
+const upgradeBtn = 'premium-badge-without-circle-icon';
+
+const clickUpgradeButton = async (page: Page) => {
+    await page.goto("https://local.studocu.com/en-gb?testRebranding=Rebranded&testRebrandingFooter=Rebranded&testRebrandingHome=Rebranded&testRebrandingAskAnExpert=Rebranded&testRebrandingNavbar=Rebranded&testRebrandingButtonsAndLinks=Rebranded&testRebrandingFonts=Rebranded&testRebrandingForms=Rebranded&testRebrandingSearchBar=Rebranded&testRebrandingTabs=Rebranded&testRebrandingSidebar=Rebranded    ");
```

Comment:
```suggestion
    await page.goto("https://local.studocu.com/en-gb?testRebranding=Rebranded&testRebrandingFooter=Rebranded&testRebrandingHome=Rebranded&testRebrandingAskAnExpert=Rebranded&testRebrandingNavbar=Rebranded&testRebrandingButtonsAndLinks=Rebranded&testRebrandingFonts=Rebranded&testRebrandingForms=Rebranded&testRebrandingSearchBar=Rebranded&testRebrandingTabs=Rebranded&testRebrandingSidebar=Rebranded");
```

It's better to use a function which generates this long irl with being able to pass some parameters to customize it.
For example we are using a package called `Routes` on FE which we just pass url's name to it and it returns full url to that page. So you can have a function to generate that url.

Also it's better to generate abtests automatically, Because they are changing most of the time and it's not good to put them statically in the url.